microservices.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Microservices
+nav_title: Microservices
+collection: microservices
+position: 2
+---
+
+# Microservices
+
+A microservice refers to a self-contained piece of functionality that can be used by a separate application to perform a very specific task. This is similar to the Service-Oriented Architecture you may be familiar with in the Enterprise Service Bus (ESB), with the critical distinction that microservices are independently maintained, written, and deployed rather than being many components within the same stack.
```

Comment:
the stuff about ESB seems to be superfluous. Maybe add a couple examples of things that would be good microservices (and as we build some out replace with live examples)